netmiko/snmp_autodetect.py

```diff
@@ -114,6 +114,11 @@
         "expr": re.compile(r"PowerConnect.*", re.IGNORECASE),
         "priority": 50,
     },
+    "mikrotik_routeros": {
+        "oid": ".1.3.6.1.2.1.1.1.0",
+        "expr": re.compile(r"^RouterOS.*", re.IGNORECASE),
```

Comment:
I would just make it be .`*RouterOS.*` (i.e. make the beginning a little more flexible). Do you want to update/change that?